docs/notes/todo.md

```diff
@@ -0,0 +1,45 @@
+# Todo
+
```

Comment:
Maybe we could link to this wiki article: https://reactos.org/wiki/Missing_ReactOS_Functionality

Reply:
Todo now gives a link to this wiki article.